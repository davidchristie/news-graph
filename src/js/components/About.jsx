import React from 'react'

import Jumbotron from './Jumbotron'

export default class About extends React.Component {
  render() {
    const title = 'About'
    return (
      <div>
        <Jumbotron title={title} />
      </div>
    )
  }
}
