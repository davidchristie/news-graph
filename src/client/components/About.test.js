/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import About from './About'

test('About renders correctly', () => {
  const wrapper = shallow(<About />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
