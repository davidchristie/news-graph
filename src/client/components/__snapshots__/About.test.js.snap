exports[`test About renders correctly 1`] = `
<div>
  <Jumbotron
    title="About" />
</div>
`;
